modules/ossm-control-plane-deploy.adoc

```diff
@@ -111,8 +111,8 @@ $ oc get smcp -n istio-system
 The installation has finished successfully when the READY column is true.
```

Comment:
By default, there is no pod "istio-policy-77fd498655-7pvjw  " and "istio-telemetry-66f697d6d5-cj28l" in a 2.0 SMCP.
2.0 SMCP Example output:
```
istio-system namespace pods: 
NAME                                    READY   STATUS    RESTARTS   AGE
grafana-77c8d875c6-glsh4                2/2     Running   0          111s
istio-egressgateway-5b87f64754-dfgkh    1/1     Running   0          112s
istio-ingressgateway-7c58799fdc-qgpjs   1/1     Running   0          114s
istiod-basic-install-cf6f7b47d-7f9bs    1/1     Running   0          3m26s
jaeger-57c7475659-5nct7                 2/2     Running   0          113s
kiali-5958d95f77-gm2tf                  1/1     Running   0          59s
prometheus-7566c7f7db-srv2t             3/3     Running   0          2m41s

# Installation result: 
NAME            READY   STATUS            PROFILES      VERSION   AGE     IMAGE REGISTRY
basic-install   9/9     ComponentsReady   ["default"]   2.0.0     3m31s   

```

Reply:
I believe we fixed this in https://github.com/openshift/openshift-docs/pull/26261 and will rebase when that PR is merged.